resources/user-agents/crawlers/search-engines/search-engines.json

```diff
@@ -474,13 +474,27 @@
           }
         },
         {
+          "match": "Mozilla/5.0 (compatible; Charlotte/*)",
+          "properties": {
+            "Browser": "Charlotte"
+          }
+        },
+        {
           "match": "Mozilla/5.0 (compatible; Charlotte/*; *)",
```

Comment:
Probably not,  I don't think either of these were covered by tests (they were both already in this file, I just moved them closer together), and they weren't catching those two Charlotte agents.

Here: https://codecov.io/gh/jaydiablo/browscap/src/5eab2770a122239d5f9c5248df0acde1b3a676cc/resources/user-agents/crawlers/search-engines/search-engines.json#L477

And here: https://codecov.io/gh/jaydiablo/browscap/src/5eab2770a122239d5f9c5248df0acde1b3a676cc/resources/user-agents/crawlers/search-engines/search-engines.json#L904

I can remove one or the other (or both).  There is a user-agent in UA Parser's test suite that these would match against:

>Mozilla/5.0 (compatible; Charlotte/1.0b; charlotte@beta.spider.com)

I can remove one of them.  Which should remain?